test/unit/specs/components/staking/TabMyDelegations.spec.js

```diff
@@ -151,7 +155,77 @@ test(`yourValidators`, () => {
         type: `AC26791624DE60`
       },
       status: 2,
-      tokens: `0`
+      tokens: `0`,
+      voting_power: `7`
     }
   ])
 })
+
+test(`address`, () => {
+  expect(
+    TabMyDelegations.computed.address({
+      user: {
+        address: `cosmos1asdfasdf`
+      }
+    })
+  ).toEqual(`cosmos1asdfasdf`)
+})
+
+test(`properties`, () => {
+  expect(
+    TabMyDelegations.computed.properties({
+      bondingDenom: `tofu-steaks`
+    })
+  ).toEqual([
```

Comment:
Good example on where to use a snapshot.

Reply:
i don't know how in @NodeGuy's brave new world.

also, the amount of unused code was _not good_.

Reply:
?

Reply:
where does the `wrapper` come from without `vuex-setup` and `mount`?

```
expect(wrapper.html()).toMatchSnapshot()
```

Reply:
there is still DOM generated in `shallow`. Components are just not resolved so they stay as `<componen-x prop1="2" />`.
In this case using a snapshot means: 
```
expect(
    TabMyDelegations.computed.properties({
      bondingDenom: `tofu-steaks`
    })
  ).toMatchSnapshot()
```

Reply:
ah cool. thank you!

Reply:
this way we don't pollute the test file with the huge json but still see if anything changes to the unexpected.